test/fitnesse/testsystems/slim/tables/SlimTableTest.java

```diff
@@ -113,8 +113,8 @@ public void replaceSymbolsFullExpansion_ShouldMatchFullSymbolName() throws Excep
   public void replaceSymbols_ShouldReplaceConcutenatedSymbols() throws Exception {
     SlimTable table = new MockTable();
     table.setSymbol("x", "1");
-    table.setSymbol("y", "1");
-    assertEquals("this is $x->[1]1 and $y->[1]1", table.replaceSymbolsWithFullExpansion("this is $x1 and $y1"));
+    table.setSymbol("y", "a");
+    assertEquals("this is $x1->[11] and $yb->[ab]", table.replaceSymbolsWithFullExpansion("this is $x1 and $yb"));
```

Comment:
Please look at issue #780.
This doesn't works. How we display it doesn't matters.
to keep the code simple it would be best to remove it.